pages/pipelines/managing_log_output.md.erb

```diff
@@ -110,3 +110,21 @@ xcodebuild <build options> | tee -a build.log | xcpretty -c
 Make sure to set the `-o pipefail` option in your buildscript as above, otherwise the build failure status might not be passed through correctly.
 
 <%= image("xcpretty.png", alt: "Screenshot of xcpretty output", size: '656x316') %>
+
+## Redacted environment variables
+
+<div class="Docs__experiment">
+<p class="Docs__experiment__heading">Experimental feature</p>
+<p>This is an experimental feature, set <code>experiment="redacted-vars"</code> in your <a href="/docs/agent/v3/configuration#experiment"> agent configuration</a> to use it.</p>
```

Comment:
@plaindocs This should be `experiment="output-redactor"`… `redacted-vars` is a separate flag for changing which env var values should be redacted.

Reply:
PR at https://github.com/buildkite/docs/pull/978